pages/view/[linkId]/index.tsx

```diff
@@ -107,6 +109,7 @@ export default function ViewPage({
     email: string;
   };
   const { data: session, status } = useSession();
+  const { authenticationCode } = router.query as { authenticationCode: string };
```

Comment:
`authenticationCode` isn't used in this component! You can remove this.

Reply:
Done